Address comments from Martin (#10)
boucadair/add-resolver-information: draft-ietf-add-resolver-info.md

   RR type and the QNAME of the domain name that is used to authenticate the
   DNS resolver (referred to as the Authentication Domain Name (ADN) in {{?I-D.ietf-add-dnr}}).

>    If the Special-Use Domain Name "resolver.arpa", as defined in

Reviewer: ```suggestion
   If the Special-Use Domain Name "resolver.arpa", defined in
```

   {{?I-D.ietf-add-ddr}}, is used to discover an encrypted DNS resolver, the
   client can retrieve the resolver information using the RESINFO RR
   type and QNAME of "resolver.arpa".  If a resolver supports DDR
   but does not support RESINFO, the client can receive a positive RESINFO

>    response from an upstream DNS resolver	or an attacker. While DNSSEC can be used as a mechanism

Reviewer: ```suggestion
   response from a legitimate upstream DNS resolver or an attacker. While DNSSEC can be considered as a candidate mechanism
```

>    response from an upstream DNS resolver	or an attacker. While DNSSEC can be used as a mechanism 

Reviewer: ```suggestion
   response from a legitimate upstream DNS resolver or an attacker. While DNSSEC can be considered as a candidate mechanism 
```

>    to protect against the attack, it's important to note that DNSSEC protection is not possible with

Reviewer: ```suggestion
   to protect against the attack, it is important to note that DNSSEC protection is not possible with
```

>    to protect against the attack, it's important to note that DNSSEC protection is not possible with 

Reviewer: ```suggestion
   to protect against the attack, it is important to note that DNSSEC protection is not possible with 
```

>    'resolver.arpa' due to the lack of uniqueness. To prevent such an attack,

Reviewer: ```suggestion
   "resolver.arpa" due to the lack of uniqueness. To prevent such an attack,
```

#  Security Considerations

To prevent DNS response forgery attacks, DNS clients using DNR MUST employ one of the following measures:

1. Establish an authenticated secure connection to the DNS resolver.

> 2. Implement local DNSSEC validation ({{Section 10 of ?RFC8499}}), to verify the authenticity of resolver information.

Reviewer: ```suggestion
2. Implement local DNSSEC validation ({{Section 10 of ?RFC8499}}) to verify the authenticity of the resolver information.
```

> 2. Implement local DNSSEC validation ({{Section 10 of ?RFC8499}}), to verify the authenticity of resolver information. 

Reviewer: ```suggestion
2. Implement local DNSSEC validation ({{Section 10 of ?RFC8499}}) to verify the authenticity of the resolver information. 
```
